analyzers/PapasSim.py

```diff
@@ -169,8 +177,8 @@ def process(self, event):
         #create history node
         #note eventually history will be created by the simulator and passed in
         # as an argument and this will no longer be needed
-        uniqueids = list(event.tracks.keys()) + list(event.ecal_clusters.keys()) + list(event.hcal_clusters.keys())
-        history =  dict( (idt, Node(idt)) for idt in uniqueids )                    
+        uniqueids = sorted(list(event.tracks.keys()) + list(event.ecal_clusters.keys()) + list(event.hcal_clusters.keys()))
```

Comment:
I don't think sorted is necessary, as the dictionary on the next line is anyway unsorted